homeassistant/components/lcn/sensor.py

```diff
@@ -103,7 +118,9 @@ def __init__(self, config: ConfigType, config_entry: ConfigEntry) -> None:
             config[CONF_DOMAIN_DATA][CONF_UNIT_OF_MEASUREMENT]
         )
 
-        self._attr_native_unit_of_measurement = cast(str, self.unit.value)
+        self._attr_native_unit_of_measurement = UNIT_OF_MEASUREMENT_MAPPING.get(
+            self.unit, None
+        )
```

Comment:
`None` is already the default of an `.get()` call.


```suggestion
        self._attr_native_unit_of_measurement = UNIT_OF_MEASUREMENT_MAPPING.get(
            self.unit
        )
```